<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_windowClass</class>
 <widget class="QMainWindow" name="main_windowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>554</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>554</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>554</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Server</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tab_widget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>551</width>
      <height>421</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="statusTip">
     <string/>
    </property>
    <property name="locale">
     <locale language="Chinese" country="China"/>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_log">
     <property name="statusTip">
      <string>Log Console</string>
     </property>
     <attribute name="icon">
      <iconset resource="Resources/icon.qrc">
       <normaloff>:/icon/disable_log.png</normaloff>
       <normalon>:/icon/disable_log.png</normalon>:/icon/disable_log.png</iconset>
     </attribute>
     <attribute name="title">
      <string/>
     </attribute>
     <attribute name="toolTip">
      <string>Log Console</string>
     </attribute>
     <attribute name="whatsThis">
      <string>Log Console</string>
     </attribute>
     <widget class="QTextBrowser" name="text_log">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>551</width>
        <height>370</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_setting">
     <property name="statusTip">
      <string>Setting</string>
     </property>
     <attribute name="icon">
      <iconset resource="Resources/icon.qrc">
       <normaloff>:/icon/disable_setting.png</normaloff>
       <normalon>:/icon/normal_setting.png</normalon>:/icon/disable_setting.png</iconset>
     </attribute>
     <attribute name="title">
      <string/>
     </attribute>
     <attribute name="toolTip">
      <string>Settings</string>
     </attribute>
     <attribute name="whatsThis">
      <string>Settings</string>
     </attribute>
     <widget class="QLabel" name="label_main_path">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>30</y>
        <width>61</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>主目录：</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_main_path">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>30</y>
        <width>31</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="Resources/icon.qrc">
        <normaloff>:/icon/file.png</normaloff>:/icon/file.png</iconset>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_port">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>60</y>
        <width>71</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>端口号：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_max_connection">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>110</y>
        <width>111</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>最大连接数：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_timeout">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>150</y>
        <width>81</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>超时时间：</string>
      </property>
     </widget>
     <widget class="QLabel" name="labe_tip_max_connection">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>110</y>
        <width>81</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>（0为不限制）</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_log_file_path">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>240</y>
        <width>101</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>日志文件位置：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_log_lines">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>190</y>
        <width>131</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>窗口最大显示行数：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="text_main_path">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>30</y>
        <width>201</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="text_port">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>60</y>
        <width>51</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="text_max_connection">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>110</y>
        <width>51</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="text_timeout">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>150</y>
        <width>51</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="text_log_max_lines">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>190</y>
        <width>51</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="text_log_path_2">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>240</y>
        <width>201</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_server_ip">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>270</y>
        <width>61</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>服务器IP：</string>
      </property>
     </widget>
     <widget class="QComboBox" name="combobox_server_ip">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>270</y>
        <width>151</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_address_family">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>300</y>
        <width>51</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>地址族：</string>
      </property>
     </widget>
     <widget class="QComboBox" name="combobox_address_family">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>300</y>
        <width>151</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_socket_stream">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>330</y>
        <width>101</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>数据传送类型：</string>
      </property>
     </widget>
     <widget class="QComboBox" name="combobox_socket_stream">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>330</y>
        <width>151</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_ipproto">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>360</y>
        <width>41</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>协议：</string>
      </property>
     </widget>
     <widget class="QComboBox" name="combobox_ipproto">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>360</y>
        <width>151</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="labe_tip_timeout">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>150</y>
        <width>81</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>(1-1000ms)</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="button_start_pause">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>420</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>运行/停止</string>
    </property>
    <property name="statusTip">
     <string>运行/停止</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="Resources/icon.qrc">
      <normaloff>:/icon/disable_running.png</normaloff>
      <normalon>:/icon/normal_running.png</normalon>
      <disabledoff>:/icon/disable_running.png</disabledoff>
      <disabledon>:/icon/normal_running.png</disabledon>:/icon/disable_running.png</iconset>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Resources/icon.qrc"/>
 </resources>
 <connections/>
</ui>
